flow-typed/Comment.js

```diff
@@ -29,6 +29,7 @@ declare type CommentSubmitParams = {
   payment_intent_id?: ?string,
```

Comment:
:small_red_triangle: Sanity-check easily circumvented

- When the following appears, just select-and-unselect a tier, and the Upload button become active again.

<img width="517" alt="image" src="https://user-images.githubusercontent.com/64950861/192421100-17a25aae-1a6f-40a8-802e-f2e42404202c.png">

Reply:
<img width="247" alt="image" src="https://user-images.githubusercontent.com/64950861/192423784-0717f8d0-250a-44a1-bfa2-44e1ee07c9c7.png">